### ISSUE TITLE
ALT-G : go back from folder not working anymore

### ISSUE THREAD

**Issue by Olifair** [opto/nostalgy-xpi#254]
The shortcut "ALT-G" with the action "go back from folder" is not working anymore, at least on my system (TB 115.7.0 Nostalgy++ 4.1.4)
When tapping ALT-G I get the same result as just tapping G : the Nostalgy input field at the bottom opens and waits for typing in a folder. 
I think that this probem is not new in the 4.1.4 version but has already existed before

Cheers

Oliver

**Comment by mdonz** [comment]
I have defined "Ctrl-G" as shortcut to go back here (TB115.7, Nostalgy 4.1.4 on Ubuntu) and that works for me.
Michael

**Comment by opto** [comment]
here (Win 11, German): G works, alt-G not, but the new ctrl-alt-G

**Comment by blerner** [comment]
I wonder if this is similar to #253 and the release comment on 4.1.3,
> this in addition uses webextention commands.
> So S is also usable as alt-S

Perhaps the handling of `Alt+...` hotkeys is more generally getting conflated with the non-`Alt+...` versions of the keys?  @Olifair does `Alt+G` work for you in 4.1.1, the way `Alt+I` works for me in v4.1.1 but not above?

**Comment by Olifair** [comment]
@blerner no, that was the whole problem in the first place, ALT-G is not working. I'm using "Q" now and that is working well :-)
and I have 4.1.4 installed.

**Comment by garnajee** [comment]
I would like to add that any of ALT+... isn't working
But with Alt**Gr**+... it works.
nostalgy++ v4.1.4
thunderbird v115.9.0

**Comment by opto** [comment]
I think TB uses a new way to grab the keypressed which I found no way to override for some letter combinations. Therefore, there are the new key combinations on the webextension side. Even there, you need to test which letter combinations are free to be used.
But once found, those should be guaranteed to work (?) by the new webextention support of TB. If those don't work, please let me know.

**Comment by opto** [comment]
actually, the old style alt-G does not work because I erroneously set alt-G in the new commands as replacement for G.

On my German PC, the new shift-alt-G for go back does not work becaue it seems to be in use by TB, but shift-alt-L works for go back.

But if I set the new alt-G to something else, the old alt-G for go back is still grabbed by TB and it does not go back. So it seems at the moment, go back needs to used with a new command

**Comment by mdonz** [comment]
FWIW, I set my "Go Back" to be Ctrl-g in the Nostalgy Preferences only (no shortcuts defined at all yet for Nostalgy in web extensions/Extension Shortcuts to avoid confusion while testing shortcuts). That's by now TB ver. 125.0.1, Ubuntu, English, Nostalgy 4.1.5.

**Comment by opto** [comment]
thanks, I can confirm this works on German Win 11 as well.

**Comment by Olifair** [comment]
I'm using just "Q" for the "go back from folder" option, and it's working well (French windows 11 system with a German keyboard). I'm so used to it by not, that probably I would keep it, even if ALT-G would work again ;-)
